### DISCUSSION TITLE
How to use the Component in another APP

### DISCUSSION THREAD

**Discussion by huseyin-bozkurt** [marcusolsson/grafana-gantt-panel#62] [Q&A] [answered]
I am going to develop a new app and i need a gantt-chart in it. Is it possible to use this plug-in in my app. If yes, how can i do that? Could you please help me about it?
Regards.

**Comment by marcusolsson** [comment, marked as answer]
Do you mean a Grafana app plugin, or an app outside of the Grafana ecosystem? While it is a React component, it's very closely tied to Grafana, so I wouldn't recommend using this outside of Grafana.

**Comment by huseyin-bozkurt** [comment]
Dear @marcusolsson , thank you very much for your response.
Yes, i was planing to use it outside of the Grafana ecosytem but after your message, i decided to find another component.
Regards.
